File: 03.business-os/PocketSecretary/iphone/PocketSecretary/BusinessOSCommonOSWrapper/BusinessOSCommonOSWrapperRegistry.swift
import Foundation

enum BusinessOSCommonOSWrapperRegistry {
    static func createDefault() -> BusinessOSCommonOSWrapperDescriptor {
        BusinessOSCommonOSWrapperDescriptor(
            appName: "PocketSecretary",
            providerRoot: "~/03.civilization-development/12.common-os",
            consumerRoot: "~/03.civilization-development/03.business-os/_commonos",
            providerRole: "shared_provider",
            consumerRole: "os_side_consumer",
            adapterModule: "adapter",
            bridgeModule: "bridge",
            mapperModule: "mapper",
            presenterModule: "presenter",
            themeModule: "theme",
            syncModule: "sync",
            uiOwner: "CommonOS",
            businessOwner: "PocketSecretary"
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/wrapper/BusinessOSCommonOSWrapperRegistry.java
package com.lsam.PocketSecretary.wrapper;

public final class BusinessOSCommonOSWrapperRegistry {

    public BusinessOSCommonOSWrapperDescriptor createDefault() {
        return new BusinessOSCommonOSWrapperDescriptor(
                "PocketSecretary",
                "~/03.civilization-development/12.common-os",
                "~/03.civilization-development/03.business-os/_commonos",
                "shared_provider",
                "os_side_consumer",
                "adapter",
                "bridge",
                "mapper",
                "presenter",
                "theme",
                "sync",
                "CommonOS",
                "PocketSecretary"
        );
    }
}
